core/os/os.h

```diff
@@ -240,6 +240,9 @@ class OS {
 
 	virtual bool is_single_window() const;
 
+	virtual void disable_xr() {}
```

Comment:
I wouldn't put this on OS. I would store the settings as a static variable on `XRServer` and add static access methods so it is nicely contained within the XR space of the application.

Reply:
Can you explain how a static method in XRServer can confine these settings to the module?

Like using call("/root/XRServer", "xr", "on") or the trick of providing function pointers that are null if the module is disabled?